src/main.rs

```diff
@@ -212,6 +221,14 @@ fn entry() -> Result<(), String> {
                     println!("{tag}");
                 }
             }
+            let error_flag_passed = matches
+                .subcommand
+                .unwrap() // Safe because we're _in_ a subcommand
+                .matches
+                .is_present(LIST_UNUSED_ERROR_OPTION);
+            if error_flag_passed && !tags_map.lock().unwrap().is_empty() {
```

Comment:
Can we add a comment justifying the `unwrap()`? You can copy the wording from line 218.